tests/test_favorites_courses.py

```diff
@@ -0,0 +1,27 @@
+from tutor import db
+from tutor.models.users import Users
+from tutor.models.course import Course
+
+
+def test_fav(client):
+    # create new user
+    u = Users(username='pypy', email='email@email.com', password='pass')
+    u.save()
```

Comment:
not really an issue, but you can use the function `create_test_user` from `user_functions.py`